src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/crushmap/crushmap.component.html

```diff
@@ -15,11 +15,10 @@
                        (updateData)="onUpdateData()">
               <ng-template #treeNodeTemplate
                            let-node>
-                <span *ngIf="node.data.status"
-                      class="badge"
-                      [ngClass]="{'badge-success': ['in', 'up'].includes(node.data.status), 'badge-danger': ['down', 'out', 'destroyed'].includes(node.data.status)}">
+                <cds-tag *ngIf="node.data.status"
+                         [class]="['tags-success', 'tags-danger'][['in', 'up'].includes(node.data.status) ? 0 : ['down', 'out', 'destroyed'].includes(node.data.status) ? 1 : 2]">
```

Comment:
can you explain this logic change ?
I am thinking previous one was rather easier to follow

Reply:
if i will keep same logic my css class is not applying.please find screenshot. 
![image](https://github.com/user-attachments/assets/e062bddd-dcf2-4972-b91c-6690769daca5)